vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -244,10 +244,10 @@ def _smart_connect(self, instance, service_check_tags):
         # Check that we have sufficient permission for the calls we need to make
         try:
             server_instance.CurrentTime()
-        except Exception:
+        except Exception as e:
             err_msg = (
-                "A connection to {} can be made, but it appears the user {} doesn't have enough permissions"
-            ).format(instance.get('host'), instance.get('username'))
+                "A connection to {} can be made, but it appears the user {} doesn't have enough permissions: {}"
+            ).format(instance.get('host'), instance.get('username'), e)
```

Comment:
I'd either catch the right set of exceptions or, since we're catching a generic one, be generic in the message as well. My point is: are we sure this will always be a permission issue?

Reply:
You're right, I'll be more generic in the error message